circle.yml

```diff
@@ -1,12 +1,9 @@
 machine:
   node:
-    version: '0.10'
+    version: '0.12'
 dependencies:
-  pre:
-    - sudo apt-get update; sudo apt-get -y install python-pip libglew-dev gdb
-    - sudo pip install awscli
```

Comment:
Is awscli installed by default on Circle VMs? It's still needed for deploying a release. If it's not preinstalled we could move the install commands into the `deployment: release` section instead.